src/components/plotly-modal/plotly-modal.scss

```diff
@@ -20,7 +20,7 @@
   top: 0;
   right: 0;
   bottom: 0;
-  left: 0;
+  left: 80px;
```

Comment:
I would suggest changing this hard-coded value to the variable we have set in the `variables` file, like we do [here](https://github.com/quantumblacklabs/kedro-viz/blob/9f0283c3f4af02c94b258311edf1997a1ad4742d/src/components/experiment-tracking/sidebar/sidebar.scss#L10).